services/venice-router/src/main/java/com/linkedin/venice/router/streaming/VeniceChunkedWriteHandler.java

```diff
@@ -67,6 +67,10 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
         return;
       }
     }
+    // check for closed channel
```

Comment:
Ignoring it silently is not good since if this issue happens, we have no info to know it is happening.
What is the issue we are trying to address here? Less exception logging?